ComponentKit/Core/Scope/CKComponentScopeHandle.h

```diff
@@ -62,8 +63,17 @@
 @property (nonatomic, readonly) CKComponentScopeHandleIdentifier globalIdentifier;
 
 /**
- Provides a responder corresponding with this scope handle. The controller will assert if called before resolution.
+ Provides a block that, when called, will return the responder associated with the "current" component generation.
  */
-- (id)responder;
+- (CKScopedResponder *)scopedResponder;
+
+/**
+ Used by the infra. Don't call unless you know what you are doing.
```

Comment:
A description of what this could be used for/how to use it would help us maintain this I think.